test/generated-pkg/app-lib.js

```diff
@@ -15,18 +15,18 @@ describe('library package', function () {
         apptype: ['empty', 'library'],
         pre: false,
         post: false,
-        license: 'LGPL-3.0',
+        license: ['LGPL-3.0', 'LGPL%20v3', 'https://www.gnu.org/licenses/lgpl-3.0'],
```

Comment:
Why do libraries default to LGPL v3? Would it be easier to have all app types use MIT by default?

Reply:
This is just a test, it doesn’t impact the default (AGPL3).I switched one test run to a different library, to make sure the actual selection is triggered during test execution.